pkg/k8s/labels.go

```diff
@@ -246,6 +246,9 @@ const (
 	// TapServiceName is the name of the tap APIService
 	TapServiceName = "linkerd-tap"
 
+	// AdmissionWebhookLabel indicates whether admission webhooks are enabled for a namespace
+	AdmissionWebhookLabel = ProxyConfigAnnotationsPrefix + "/admission-webhooks"
```

Comment:
TIOLI, or a separate PR: You could update `namespace.yaml`, `proxy-injector-rbac.yaml` and `sp-validator-rbac.yaml`  with this new constant...